images/aleph/flake.nix

```diff
@@ -8,7 +8,7 @@
   inputs = {
     nixpkgs.url = "github:nixos/nixpkgs/nixos-24.11";
     nixpkgs-unstable.url = "github:nixos/nixpkgs/nixos-unstable";
-    jetpack.url = "github:anduril/jetpack-nixos/master";
```

Comment:
Can this still refer to master? Since the lock file references the commit anyways.

Reply:
I was having trouble forcing an update to the latest version with nix flake update, but I can try and set it back to master to see if it still works